Dockerfile

```diff
@@ -9,5 +9,5 @@ ADD scm-source.json /scm-source.json
 EXPOSE 8080
 
 # run the server when a container based on this image is being run
-ENTRYPOINT java -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
+ENTRYPOINT java $JAVA_OPTS -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
```

Comment:
@rcillo Not exactly. Jvm doesn't check for JAVA_OPTS environment variable. It is usually done by scripts that run jvm (in tomcat, ant etc.)
At least this is what I've read on stackoverflow and my local test confirmed that.